src/framework/components/rigid-body/system.js

```diff
@@ -181,16 +222,6 @@ class RigidBodyComponentSystem extends ComponentSystem {
 
         this.schema = _schema;
 
-        this.maxSubSteps = 10;
-        this.fixedTimeStep = 1 / 60;
-        this.gravity = new Vec3(0, -9.81, 0);
-
-        // Arrays of pc.RigidBodyComponents filtered on body type
-        this._dynamic = [];
-        this._kinematic = [];
-        this._triggers = [];
-        this._compounds = [];
-
         this.collisions = {};
         this.frameCollisions = {};
```

Comment:
Should these two objects also move up or wait for Record type?

Reply:
They could go up, yes, but the value is another object type and since it's an internal property, I thought I'd leave it for now.